components/pages/explorer/QueryActions/ShareButton.tsx

```diff
@@ -24,8 +46,47 @@ const ShareButton = ({
   title?: string;
 }) => {
   const theme: DefaultTheme = useTheme();
+  const copyLink = useCallback(() => navigator.clipboard.writeText(queryUrl), [queryUrl]);
 
   const shareChannels = [
+    <Button
```

Comment:
this button is not the uikit's that allows variations. 
will abstract it into a reusable component if we ever need another faceless button